tests/python/gaia-ui-tests/gaiatest/tests/graphics/RTL/test_Contacts_RTL.py

```diff
@@ -0,0 +1,41 @@
+import pdb
+from gaiatest.gaia_graphics_test import GaiaImageCompareTestCase
+from gaiatest.apps.contacts.app import Contacts
+from gaiatest.apps.contacts.regions.settings_form import SettingsForm
+from gaiatest.apps.contacts.regions.contact_form import ContactForm
+
+class TestContactsRTL(GaiaImageCompareTestCase):
+   
+    def test_contacts_app(self):
+
+        contacts = Contacts(self.marionette)
+        contacts.launch()
+        self.take_screenshot('main')
```

Comment:
For each screenshot command, you can specify where you are- instead of using 'main' you could say 'contacts' , 'contacts-import', 'contacts-sim_import', 'contacts-orderby_lastname' ) then it would be more clear to the user since this will be included in the image filename.